core/loaders/load_yaml.cpp

```diff
@@ -12,61 +12,75 @@
 //
 //  ***********************************************************************************************
 
+#include "fastyamlloader.h"
 #include "core/raw/rawfile.h"
 #include "qcr/engine/debug.h"
-#include "yaml-cpp/include/yaml-cpp/yaml.h"
 
 namespace  {
 
-void readInstrument(const YAML::Node& node, Rawfile& rawfile)
+typedef loadYAML::YamlNode YamlNode;
+
+template <class Container, class Function>
+auto map (const Container &cont, Function fun) {
```

Comment:
function name 'map' is a bit unspecific

and please, for each new function a one-line doxygen comment that tells us the purpose of the function